src/main/java/runner/Storage.java

```diff
@@ -89,4 +80,18 @@ public void saveList() {
             System.out.println(e.getMessage());
         }
     }
+
+    /**
+     * Check the status of the Task and add it to the TaskList.
+     * @param line Description of a Task.
+     * @param t Task checked.
+     */
+    public void check_add(String line, Task t) {
```

Comment:
Just a minor stylistic issue. You can use camelCase for "check_add" to something like "addCheckedTask".